### PR TITLE
Change phoronix-to-OpenForBC converter according to actual OpernForBC benchmark interface.

### PR DESCRIPTION
**Pull request by AlessioBorriero**
Change phoronix-to-OpenForBC converter according to actual OpenForBC  benchmark interface.

### CODE REVIEW

**Comment by gabrielefronze** [comment]
Ciao @AlessioBorriero! I've added a review, can you please add a description to the PR in the submission message as well? Thanks!

**Comment by gabrielefronze** [comment]
@AlessioBorriero I've added flake8 CI, have a look at what should be fixed by running flake8 locally. Thanks!

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    --hash=sha256:4ca091dea149f945ec56afb48dae714f21e8692ef22a395223bcd328961b6a0e
+wcwidth==0.2.5 ; python_version >= "2.7" \
+    --hash=sha256:beb4802a9cebb9144e99086eff703a642a13d6a0052920003a230f3294bbe784 \
+    --hash=sha256:c4d647b99872929fdb7bdcaa4fbe7f01413ed3d98077df798530e5b04f116c83
+yaspin==2.1.0 ; python_full_version >= "3.6.2" and python_full_version < "4.0.0" \
+    --hash=sha256:d574cbfaf0a349df466c91f7f81b22460ae5ebb15ecb8bf9411d6049923aee8d \
+    --hash=sha256:c8d34eca9fda3f4dfbe59f57f3cf0f3641af3eefbf1544fbeb9b3bacf82c580a
+zipp==3.6.0 ; python_version >= "3.6" and python_version < "3.8" \
+    --hash=sha256:9fe5ea21568a0a70e50f273397638d39b03353731e6cbbb3fd8502a33fec40bc \
+    --hash=sha256:71c644c5369f4a6e07636f0aa966270449561fcea2e3d6747b8d23efaa9d7832
```
[the remark this answers is not in the document]
Comment (gabrielefronze) [reply requirements.txt:84]: To be discussed.

File: phoronix_parser.py [hunk 2]
```diff
@@ -302,8 +307,49 @@ def create_info_file(target_dir, test_definition_xml, default_settings_file):
...
+
+            stat = stat.replace("#_RESULT_#", regex)   
+            mini_dict = {}
+            mini_dict['regex'] = stat
+            results_dict[stat_name] = mini_dict
+            results_string = json.dumps(results_dict)        
+
+    file_inplace_replace(file_path=target_benchmark_info_file,
+                        search_string="PUT_STATS_HERE",
+                        replace_string=results_string)
```
[the remark this answers is not in the document]
Comment (AlessioBorriero) [reply phoronix_parser.py]: Yes, I did. I look a quite big sample of different benchamrk and in each of them only one of these two tags is used.

[the remark this answers is not in the document]
Comment (gabrielefronze) [reply phoronix_parser.py]: Roger that!

File: phoronix_benchmark.json.template [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+    "name": "PUT_NAME_HERE",
+    "description": "PUT_DESCRIPTION_HERE",
+    "default_preset": "PUT_DEFAULT_PRESETS_HERE",
+    "setup_command": "./setup.sh",
+    "run_command": "PUT_RUN_COMMAND_HERE",
+    "test_command": ["pip install flake8", "flake8 --exclude .venv"],
+    "stats": 
+        PUT_STATS_HERE,
+    "virtualenv": true   
+}
```
[the remark this answers is not in the document]
Comment (AlessioBorriero) [reply phoronix_benchmark.json.template]: Fixed!

File: phoronix_parser_cli.py [hunk 4]
```diff
@@ -1,11 +1,15 @@
...
+import subprocess
 import argparse
 import sys
 from phoronix_parser import installer_map, phoronix_init, phoronix_list, phoronix_install
 
 
 if __name__ == "__main__":
+    bash_command = "pip install -r requirements.txt"
+    subprocess.Popen(bash_command.split() ,stdout=subprocess.PIPE)
+
```
[the remark this answers is not in the document]
Comment (AlessioBorriero) [reply phoronix_parser_cli.py:9]: Fixed, I removed these two lines from the file.

File: phoronix_setup.sh.template [hunk 5]
```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+sed -i '2 i LOG_FILE=/dev/stdout' BENCHMARK_NAME
```
[the remark this answers is not in the document]
Comment (AlessioBorriero) [reply phoronix_setup.sh.template]: The line above is necessary in order to define the variable LOG_FILE inside the benchmark script. Without this command the variable would remain undefined.

**Pull request merged**